spec/features/guide_spec.rb

```diff
@@ -236,6 +236,30 @@
     end
   end
 
+  describe "slug generation" do
+    it "generates slug", js: true do
+      {
+        "Guide title": "guide-title",
+        "slug--with-----hyphens": "slug-with-hyphens",
+        "       space    slugs  ": "space-slugs",
```

Comment:
whatcha gonna do
whatcha gonna do when we come for you
![](http://cdn.collider.com/wp-content/uploads/2015/08/bad-boys.jpg)